docs/spring-boot/app-starter.md

```diff
@@ -126,3 +127,23 @@ public class ExtAppSpringApplication {
 By configuring the property `bdk-app.auth.enabled=true`, the Application backend will provide Apis for performing
 the [Circle of Trust](https://docs.developers.symphony.com/building-extension-applications-on-symphony/app-authentication/circle-of-trust-authentication) of Symphony:
 [Circle of Trust API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/SymphonyPlatformSolutions/symphony-api-client-java/master/docs/spring-boot/circle-of-trust.yaml)
+
+
+## Application health status
+The starter exposes the core components health status through the actuator endpoint from Spring Boot, for instance,
+http://localhost:8080/${root_path}/actuator/health/, which gives a global application health status, or
+http://localhost:8080/${root_path}/actuator/health/symphonyBdk, which specifically gives the Symphony components
+health status only.
+
+By default, the components health details are not exposed by Spring Boot. In order to see the details, the following
+configuration must set in the `application.xml`
+
+```
+management:
+  endpoint:
+    web:
+      exposure:
+        include: 'symphonyBdk'
+    health:
+      show-details: "ALWAYS"
+```      
```

Comment:
we need maybe to update configuration.md too with this new added config: https://github.com/finos/symphony-bdk-java/blob/main/docs/configuration.md

Reply:
Just check this configuration.md content, il looks like it s more like BDK configuration only, if we put this into configuration.md, i feel that it s kinda mix between BDK and spring boot config, especially considering that this config is in app-starter only.